.eslintrc

```diff
@@ -14,6 +14,10 @@
     // A jsx extension is not required for files containing jsx
     "react/jsx-filename-extension": 0,
     // This rule struggles with flow and class properties
-    "react/sort-comp": 0
+    "react/sort-comp": 0,
+    // ignore linebreak style. the CRLF / LF endings wont matter
+    // if a windows user correctly converts CRLF to LF upon commits otherwise
+    // there are errors every line.
+    "linebreak-style": 0
```

Comment:
Windows needs to conform to the standards of the other two superior platforms :)